lib/aamva/request/security_token_request.rb

```diff
@@ -27,9 +28,11 @@ def nonce
       end
 
       def send
-        Response::SecurityTokenResponse.new(
-          http_client.post(url, body, headers),
-        )
+        with_retries(max_tries: 2, rescue: [Faraday::TimeoutError, Faraday::ConnectionFailed]) do
```

Comment:
Nice, this is a new one for me

Reply:
it's a really great gem, it also does like exponential backoff, jittering, all nice things and simplifies code a ton